test/auth.js

```diff
@@ -223,5 +169,47 @@ describe('Authentication', () => {
                 done();
             })
         })
+        it('should fail because the token verion is invalid', (done) => {
+            let refreshToken = createRefreshToken("testmail123@test.de", 2)
+            chai.request(server)
+            .post('/refresh_token')
+            .set('Cookie', `jid=${refreshToken}`)
+            .send()
+            .end((err, res) => {
+                res.should.have.status(401);
+                res.body.should.have.property("message").eql("Invalid refresh token version");
+                done();
+            })
+        })
+    })
+    describe('/Logout user', () => {
+        it('should fail because the jwt is invalid', (done) => {
+            let loginData = {
+                "email": "testmail@test.de",
+                "password": "Test_pass1!",
+            }
+            chai.request(server)
+            .post('/logout')
```

Comment:
It says `JsonWebTokenError: jwt must be provided` because [coming token is null or empty](https://stackoverflow.com/questions/48476625/having-trouble-with-jsonwebtoken-jsonwebtoken-error-jwt-must-be-provided) but I can see revokeRefreshToken controller in auth.ts that it gives token properly.. ?_?

Is this because proper token isn't given in this test block? I see a mocked token right below but it's still not null or empty so this error shouldn't happen.. care to elaborate? can't understand what's going on :D 

btw test just works fine on this one fwiw

![image](https://user-images.githubusercontent.com/38537908/115058930-16602b00-9ee6-11eb-8132-dc327196cced.png)

Reply:
#31